core/client/app/controllers/settings/general.js

```diff
@@ -39,6 +51,12 @@ export default Controller.extend(SettingsSaveMixin, {
         return this.get('model.cover') || '';
     }),
 
+    localTime: computed('selectedTimezone', 'clock.second', function () {
+        let offset = this.get('selectedTimezone.name');
+        this.get('clock.second');
```

Comment:
Is this `get('clock.second')` doing anything here? I think it should be safe to remove as we're not storing the value or using it anywhere else.

Would renaming the `offset` variable to `timezone` here better reflect it's value?